package.json

```diff
@@ -35,11 +35,12 @@
     "@nuxtjs/eslint-config": "^6.0.1",
     "@nuxtjs/eslint-module": "^3.0.2",
     "@nuxtjs/style-resources": "^1.2.1",
+    "@supabase/supabase-js": "^2.8.0",
     "@testing-library/vue": "^5.8.3",
     "@vue/test-utils": "^1.3.3",
     "@vue/vue2-jest": "^27.0.0-alpha.2",
     "babel-core": "^7.0.0-bridge.0",
-    "babel-jest": "^29.4.2",
+    "babel-jest": "^",
```

Comment:
Is it the best way to not to specify any version ?

Reply:
resolved